### PR TITLE
Adjust Compass documentation in official Kyma docs to the new "separate Compass" approach

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Changes proposed in this pull request:

- Rename the `compass` topic to `runtime-agent`
- Remove Compass-related documentation and assets
- Remove references to the official Compass docs

**Related issue(s)**

### CODE REVIEW

File: docs/runtime-agent/02-01-runtime-agent-workflow.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+title: Architecture
 ---
 
-This document presents the workflow of the Runtime Agent. 
+This document presents the workflow of the Runtime Agent.
 
 ![Runtime Agent architecture](./assets/runtime-agent-architecture.svg)
 
-1. Runtime Agent fetches the certificate from the Connector to initialize connection with Compass.
+1. Runtime Agent fetches the certificate from the [Connector](https://github.com/kyma-incubator/compass/blob/master/docs/connector/02-01-connector-service.md) to initialize connection with Compass.
```
Comment (majakurcius) [inline docs/runtime-agent/02-01-runtime-agent-workflow.md:9]: I was thinking: do we want to make the `initialize connection with Compass` bit to link to either [`03-01-connection-with-compass.md`](https://github.com/kyma-project/kyma/blob/007037bd803ee50badf576d61a61360cdbb33271/docs/runtime-agent/03-01-connection-with-compass.md) or [`08-07-establish-secure-connection-with-compass.md`](https://github.com/kyma-project/kyma/blob/007037bd803ee50badf576d61a61360cdbb33271/docs/runtime-agent/08-07-establish-secure-connection-with-compass.md)? Just a thought.

File: docs/runtime-agent/08-07-establish-secure-connection-with-compass.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ To establish a secure connection with Compass and generate the client certificat
 ## Prerequisites
 
 - [OpenSSL toolkit](https://www.openssl.org/docs/man1.0.2/apps/openssl.html) to create a Certificate Signing Request (CSR), keys, and certificates which meet high security standards
-- Compass (version 1.8 or higher)
+- [Compass](https://github.com/kyma-incubator/compass) (version 1.8 or higher)
```
Comment (majakurcius) [inline docs/runtime-agent/08-07-establish-secure-connection-with-compass.md:11]:
```suggestion
- [Compass](https://github.com/kyma-incubator/compass)
```

We no longer have Compass versioning.

File: docs/runtime-agent/08-08-maintain-secure-connection-with-compass.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ After you have established a secure connection with Compass, you can fetch the c
 ## Prerequisites
 
 - [OpenSSL toolkit](https://www.openssl.org/docs/man1.0.2/apps/openssl.html) to create a Certificate Signing Request (CSR), keys, and certificates which meet high security standards
-- Compass (version 1.8 or higher)
+- [Compass](https://github.com/kyma-incubator/compass) (version 1.8 or higher)
```
Comment (majakurcius) [inline docs/runtime-agent/08-08-maintain-secure-connection-with-compass.md:11]:
```suggestion
- [Compass](https://github.com/kyma-incubator/compass)
```

File: docs/runtime-agent/08-09-revoke-client-certificate.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ After you have established a secure connection with Compass and generated a clie
 ## Prerequisites
 
 - [OpenSSL toolkit](https://www.openssl.org/docs/man1.0.2/apps/openssl.html) to create a Certificate Signing Request (CSR), keys, and certificates which meet high security standards
-- Compass (version 1.8 or higher)
+- [Compass](https://github.com/kyma-incubator/compass) (version 1.8 or higher)
```
Comment (majakurcius) [inline docs/runtime-agent/08-09-revoke-client-certificate.md:13]:
```suggestion
- [Compass](https://github.com/kyma-incubator/compass)
```

File: docs/runtime-agent/08-10-runtime-agent-configuration.md [hunk 5]
```diff
@@ -7,9 +7,9 @@ This tutorial shows how to configure the Runtime Agent with Compass.
 
 ## Prerequisites
 
-- Compass (version matching the Runtime Agent)
+- [Compass](https://github.com/kyma-incubator/compass) (version matching the Runtime Agent)
```
Comment (majakurcius) [inline docs/runtime-agent/08-10-runtime-agent-configuration.md:10]:
```suggestion
- [Compass](https://github.com/kyma-incubator/compass)
```

File: docs/runtime-agent/08-11-runtime-agent-reconnecting.md [hunk 6]
```diff
@@ -7,7 +7,7 @@ This tutorial shows how to reconnect the Runtime Agent with Compass after the es
 
 ## Prerequisites
 
-- Compass (version matching the Runtime Agent)
+- [Compass](https://github.com/kyma-incubator/compass) (version matching the Runtime Agent)
```
Comment (majakurcius) [inline docs/runtime-agent/08-11-runtime-agent-reconnecting.md:10]:
```suggestion
- [Compass](https://github.com/kyma-incubator/compass)
```

**Comment by kazydek** [comment]
@klaudiagrz Does it mean Runtime Agent can now be added to the [overall component list](https://kyma-project.io/docs/#details-components) and to the [component installation table](https://kyma-project.io/docs/master/root/kyma/#installation-overview)?

**Comment by klaudiagrz** [comment]
@kazydek Yes, thank you for the reminder! 👍

File: docs/runtime-agent/02-01-runtime-agent-workflow.md (hunk 1 above)
Comment (klaudiagrz) [reply docs/runtime-agent/02-01-runtime-agent-workflow.md:9]: I'm afraid it will be too packed with links if we do so 😕

**Pull request merged**